src/gnomeshellsearch.py

```diff
@@ -95,19 +96,16 @@ def __init__(self, notebook=None, search_all=True):
         self.search_all = search_all
 
     def main(self):
-        import gtk
-
-        gtk.main()
+        GLib.MainLoop().run()
 
     def quit(self):
```

Comment:
This here apparently is not used and is not part of dbus [Object class](https://dbus.freedesktop.org/doc/dbus-python/dbus.service.html#dbus.service.Object).